sklearn/pipeline.py

```diff
@@ -1506,7 +1506,7 @@ def make_pipeline(*steps, memory=None, transform_input=None, verbose=False):
     )
 
 
-def _transform_one(transformer, X, y, weight, params=None):
+def _transform_one(transformer, X, y, weight, params=Bunch(transform={})):
```

Comment:
For our scikit-learn's case, I think `params` is always passed in, so I think we can safely remove the default here:

```suggestion
def _transform_one(transformer, X, y, weight, params):
```

Reply:
That works for me. It would still have broken my use case, but it would have made it much easier to debug.
Proposed changes committed.